### PR TITLE
update reva to v1.12

### PR DESCRIPTION
**Pull request by C0rby**
Updating reva for the upcoming release.
This is just a preparation, later today when reva has been tagged I'll update to the tagged version.

- [x] Change reva version to 1.12

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Core-API-Tests-ocis-storage-8</strong> failed. The build is cancelled...</summary>

**Comment by phil-davis** [comment]
We got the core commit id for tests up-to-date in PR #2419 
Rebasing this will be a good idea, or you might get out-dated expected-failures diffs.

**Comment by phil-davis** [comment]
https://github.com/owncloud/ocis/tree/update-reva says it is 6 commits behind master. That is before web 4.1.0 was merged to master. It might be worth rebasing this so it runs with the latest web.

File: changelog/unreleased/update-reva.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Enhancement: Update reva to v1.12
```
Comment (pascalwengerter) [inline changelog/unreleased/update-reva.md:1]: Misses the `change` entry from https://github.com/cs3org/reva/pull/1982 if I'm not mistaken? 🤔

Comment (C0rby) [reply changelog/unreleased/update-reva.md:1]: I'm only listing the changes which are relevant for oCIS. I don't think copying all entries from the reva changelog to ocis makes much sense to be honest.

Comment (pascalwengerter) [reply changelog/unreleased/update-reva.md:1]: Got it, thanks!

**Review by pascalwengerter — APPROVED** [review]
LGTM

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Pull request merged**